docs/troubleshooting.md

```diff
@@ -35,3 +36,28 @@ using Minishift.
   systemctl start virtlogd
   systemctl enable virtlogd
   ```
+
+
+## XHYVE drivers
+
+- Could not create vmnet interface, permission denied or no entitlement
+
```

Comment:
There is no clear separation of where the problem is being stated and where the solution is beginning. This become more in [KVM driver section](https://github.com/praveenkumar/minishift/blob/05b42f4cd8f505dc2e58a0d54564fe86985a576b/docs/troubleshooting.md#kvm-drivers) where the solution start with small caps. 

I suggest to add full stop or `?` to end the problem statement and use Capital letter to begin the solution